modular_skyrat/modules/cryosleep/code/admin.dm

```diff
@@ -0,0 +1,22 @@
+// Send player in not-quiet cryopod. If with_paper = TRUE, place a paper with notification under player.
+/mob/living/carbon/human/proc/send_to_cryo(with_paper = FALSE) 
```

Comment:
Does it really have to be just humans? Couldn't we just make it be at `/mob` level (if any mob can be sent to cryo)?

Otherwise, that will still mean we won't be able to send silicons to cryo using this, nor sentient simple mobs (albeit I don't know if they could cryo to begin with)